CHANGES.md

```diff
@@ -33,6 +33,11 @@ OpenSSL 3.2
 
 ### Changes between 3.1 and 3.2 [xx XXX xxxx]
 
+ * The BLAKE2b and BLAKE2s hash algorithms support a configurable output length
+   by setting the "size" parameter.
+
+   *Čestmír Kalina, Tomáš Mráz, and Ahelenia Ziemiańska*
+
```

Comment:
Unfortunately this needs two entries. One in `### Changes between 3.1 and 3.2` for BLAKE2b and another in changes between 3.2 and 3.3 for BLAKE2s. Also the first one needs to be in a separate PR so it gets merged to the 3.2 branch as well.

Reply:
does it? I only see 786b9a8d3f8e203c5536e36b9a9bab83bde0311a in `openssl-3.2.0-{alpha{1,2},beta1}` tags, and it doesn't look backported into `openssl-3.1.4` under another SHA either?

Reply:
yep, misread and misunderstood. fixed